.github/workflows/enforceTestCoverage.yml

```diff
@@ -29,22 +29,40 @@ jobs:
       MIN_TEST_COVERAGE: ${{ secrets.MIN_TEST_COVERAGE }}
     steps:
       - uses: actions/checkout@v4.1.7
+        with:
+          fetch-depth: 0
+
+      - name: Check for Solidity file changes
+        id: check_files
+        run: |
+          git fetch origin ${{ github.base_ref }} --depth=1
+          CHANGED_FILES=$(git diff --name-only origin/${{ github.base_ref }}...HEAD | grep -E '^src/.*\.sol|^test/.*\.sol' || true)
```

Comment:
I personally feel it's nice and helpful to add more comments.....e.g. for the fact that now the git action will only check for changes in folders src/ and test/.
This could be a comment at the beginning of the file or in the code itself. 

Please try to apply extensive commenting on all work you do.

We can merge this PR for now though, think we have to touch this git action again sometime soon.